File: src/main/java/org/example/EjerJava/Ej10.java
package org.example.EjerJava;

import java.util.Scanner;

public class Ej10 {
    public static void main(String[] args) {
        System.out.println("Convertir Celsius a Fahrenheit o Fahrenheit a Celsius? (c / f): ");
        Scanner sc = new Scanner(System.in);
        String respuesta = sc.nextLine();

        if (respuesta.equals("c")){
            System.out.println("Temperatura en grados Celsius: ");

            String temp = sc.nextLine();
            float celsius = Float.parseFloat(temp);

            float fahrenheit = (celsius * 9/5) + 32;

            System.out.println("Temperatura en Fahrenheit: " + fahrenheit);

        }else if (respuesta.equals("f")){
            System.out.println("Temperatura en grados Fahrenheit: ");

            String temp = sc.nextLine();
            float fahrenheit = Float.parseFloat(temp);

            float celsius = (fahrenheit - 32 ) +  5/9;

            System.out.println("Temperatura en Celsius: " + celsius);
        }
    }
}

File: src/main/java/org/example/ejerKotlin/Ej10.kt
package org.example.ejerKotlin

fun main (args: Array<String>) {
    print("Convertir Celsius a Fahrenheit o Fahrenheit a Celsius? (c / f): ")
    val respuesta = readln()

    if (respuesta == "c") {
        print("Temperatura en grados Celsius: ")
        val celsius= readln().toFloat()
        val fahrenheit = (celsius * 9/5) + 32

        print("Temperatura en Fahrenheit: $fahrenheit")

    }else if (respuesta == "f") {
        print("Temperatura en grados Fahrenheit: ")
        val fahrenheit= readln().toFloat()
        val celsius = (fahrenheit - 32) + 5/9

        print("Temperatura en Fahrenheit: $celsius")
    }

}